### PR TITLE
fix: version, release and build information PT-1652

### PR DESCRIPTION
**Pull request by terovirtanen**
### Closes :no_good_woman:

**[PT-1652](https://helsinkisolutionoffice.atlassian.net/browse/PT-1652)**

## Additional notes :spiral_notepad:

[PT-1652]: https://helsinkisolutionoffice.atlassian.net/browse/PT-1652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**kultus-ui branch is deployed to platta: https://kultus-ui-pr284.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-ui branch is deployed to platta: https://kultus-ui-pr284.dev.hel.ninja** :rocket::rocket::rocket:

File: src/pages/api/__tests__/version.test.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    };
+  });
+  const res = {
+    status,
+  };
+
+  handler(req, res);
+  const res_json = json.mock.calls[0][0];
+
+  // console.log(res_json);
```
Comment (karisal-anders) [inline src/pages/api/__tests__/version.test.ts:18]: Leftover debug code? It should be removed IMHO.

**Review by melniiv — APPROVED** [review]

File: src/pages/api/__tests__/version.test.ts (hunk 1 above)
Comment (terovirtanen) [reply src/pages/api/__tests__/version.test.ts]: removed

**Comment by terovirtanen** [comment]
**kultus-ui branch is deployed to platta: https://kultus-ui-pr284.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-ui branch is deployed to platta: https://kultus-ui-pr284.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**